### PR TITLE
Add local and config

### LINKED ISSUE #83: Make a config file
Make a config file for things such as
- powersave at battery percent
- performance at batter percent
- powersave at battery percent when charging
- performance at batter percent when charging
- default to governor

```toml
# This will keep the powersave on at all times when the battery is under 40, unless manually set
powersave_under = 40

# This will set powersave if the battery is charging and is under 30 to charge faster
charging_powersave_under = 30
```

### PR DESCRIPTION
**Pull request by JakeRoggenbuck**
fixes #83 

## TODO
- [x]  Check config directory
- [x]  Make config directory
- [x] Make config file
- [x] Use default config if file not found

### CODE REVIEW

File: src/power.rs [hunk 1]
```diff
@@ -73,13 +71,11 @@ pub fn read_battery_charge() -> Result<i8, Error> {
     let battery_charge_path: Vec<&str> = vec![
         "/sys/class/power_supply/BAT/capacity",
         "/sys/class/power_supply/BAT0/capacity",
-        "/sys/class/power_supply/BAT1/capacity"
+        "/sys/class/power_supply/BAT1/capacity",
```
Comment (Shuzhengz) [inline src/power.rs:74]: maybe add a BAT2 as well just in case? ( i do have a BAT2 in my system but its empty so idk what happened there)

Comment (JakeRoggenbuck) [reply src/power.rs:74]: yea

**Comment by Camerooooon** [comment]
Panicked `thread 'main' panicked at 'No such file or directory (os error 2)', src/config.rs:24:19
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace`

**Comment by JakeRoggenbuck** [comment]
Oh yea, thanks for catching that, i forgot to make something to check if the file exists. It check directory, just not file. I'll add that, thanks!

**Comment by JakeRoggenbuck** [comment]
@Camerooooon I think I fixed it

**Comment by JakeRoggenbuck** [comment]
image

**Review by Shuzhengz — APPROVED** [review]
lgtm

**Pull request merged**